src/main/java/com/andreamazzarella/contact_manager/Age.java

```diff
@@ -1,36 +1,33 @@
 package com.andreamazzarella.contact_manager;
 
-import java.util.regex.Pattern;
-
 public class Age {
```

Comment:
thanks, I like `inYears` a lot better than `toInt`.

I wonder if I should also change the accessor for `TelephoneNumber` form `toString` to something else? I cannot think of anything better though..